src/hotspot/os/posix/include/jvm_md.h

```diff
@@ -37,13 +37,6 @@
 #include <stddef.h>             /* For ptrdiff_t */
 #include <stdint.h>             /* For uintptr_t */
 
-#define JNI_ONLOAD_SYMBOLS   {"JNI_OnLoad"}
-#define JNI_ONUNLOAD_SYMBOLS {"JNI_OnUnload"}
```

Comment:
are these changes related to the Windows 32-bit x86 port?

Reply:
After removal of Windows 32-bit x86 port, all definitions of these macros are identical, so are merged into jvm.h.
There is additional followup work involving these: see https://bugs.openjdk.org/browse/JDK-8343703.